ethereum/statetransition/src/testFixtures/java/tech/pegasys/teku/statetransition/BeaconChainUtil.java

```diff
@@ -193,7 +179,11 @@ public SignedBeaconBlock createAndImportBlockAtSlot(
               + ": "
               + block);
     }
-    forkChoice.processHead();
+
+    final StoreTransaction tx = recentChainData.startStoreTransaction();
+    tx.updateHead();
+    tx.commit().reportExceptions();
```

Comment:
nit: Probably better to use `.join()` or assert it's completed.  `reportExceptions` will allow the test to continue even if the commit fails.

```suggestion
    assertThat(tx.commit()).isCompleted();
```